selector.go

```diff
@@ -1,7 +1,25 @@
 package myradio
 
 import (
-	"github.com/UniversityRadioYork/myradio-go/api"
+//"github.com/UniversityRadioYork/myradio-go/api"
+)
+
+const (
```

Comment:
This was basically copied from the PHP-side implementation. Thought about having separate methods to check these properties but that would've been even more messy. Other suggestions welcome.